Google.Api.Gax.Grpc/ClientBuilderBase.cs

```diff
@@ -441,14 +451,21 @@ protected virtual GrpcChannelOptions GetChannelOptions()
         /// Returns whether or not a channel pool can be used if a channel is required. The default behavior is to return
         /// true if and only if no quota project, scopes, credentials or token access method have been specified. Derived classes should
         /// override this property if there are other reasons why the channel pool should not be used.
+        /// We do not enable self-signed Jwts with channel pools until credentials are not part of the key
```

Comment:
This is an implementation detail that we don't need to include on the doc.

I would change two senteces up to say something like:

/// true if and only if no quota project, scopes, credentials or token access method have been specified and UseJwt... == GetChannelPool().UseJwt....

Reply:
We have to remember that ourselves then :) Ok.